rotkehlchen/chain/ethereum/modules/eth2/eth2.py

```diff
@@ -179,6 +179,7 @@ def get_balances(
     def get_details(
             self,
             addresses: list[ChecksumEvmAddress],
+            validator_indices: Optional[set[int]] = None,
```

Comment:
Why do you need to call and modify this? We discussed in the very beginning of this feature that we can't have the beaconchain api code run each time the filter changes. This needs to be split into different code.

And filtering should get out of this function